### PR TITLE
Copy and Rename Dataframe Right-clicks

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

File: instat/dlgRenameSheet.vb [hunk 1]
```diff
@@ -31,7 +31,7 @@ Public Class dlgRenameSheet
     End Sub
 
     Private Sub ReopenDialog()
-
+        SetDefaults()
```
Comment (dannyparsons) [inline instat/dlgRenameSheet.vb]: We shouldn't be setting defaults every time the dialog is opened. If there's code needed here than separate it out from SetDefaults()

Comment (stevenndungu) [inline instat/dlgRenameSheet.vb]: OK, reset of the data frame is the one that helps. done that.

File: instat/dlgRenameSheet.vb [hunk 2]
```diff
@@ -31,7 +31,8 @@ Public Class dlgRenameSheet
     End Sub
 
     Private Sub ReopenDialog()
-        SetDefaults()
+        'Reseting ucrDataFrame to ensure that it displays the current data frame on the grid 
+        ucrDataFrameToRename.Reset()
```
Comment (dannyparsons) [inline instat/dlgRenameSheet.vb]: OK I think we will change how this works later but we can keep it for now.

**Comment by stevenndungu** [comment]
Ok, nice.

**Pull request merged**